ysoserial/Helpers/YamlDotNet.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+// Coded by Soroush Dalili (@irsdl)
+namespace ysoserial.Helpers
+{
+    class YamlDotNet
+    {
+        public static String Minify(String xmlDocument)
+        {
+            xmlDocument = Regex.Replace(xmlDocument, "[\r\n]", "");
```

Comment:
Is this removing new lines in yaml files? is the output still valid?

Reply:
I tried it using the following command and it still works:
```
.\ysoserial.exe -g ObjectDataProvider -f YamlDotNet -c calc -t  --minify
```
So I think this is right for YamlDotNet formatter.

Reply:
ok, weird though